src/main/java/org/elasticsearch/app/Harvester.java

```diff
@@ -1611,41 +1606,66 @@ private void harvest(QueryExecution qExec) {
      * and harvests the results of the query.
      */
     private void harvestFromEndpoint() {
-
         logger.info("Harvest from endpoint ---------------------------------------------------------------");
-        Query query;
-        QueryExecution qExec;
+        Model model = ModelFactory.createDefaultModel();
+        ExecutorService threadpool = Executors.newCachedThreadPool();
+        List<Future<Model>> futureTasks = new ArrayList<>();
+        Integer queryNumber = 0;
 
         for (String rdfQuery : rdfQueries) {
             if (closed) return;
 
             logger.info(
-                    "Harvesting with query: [{}] on index [{}] and type [{}]",
-                    rdfQuery, indexName, typeName);
+                    "Harvesting {}. query on index [{}] and type [{}]",
+                    ++queryNumber, indexName, typeName);
```

Comment:
Increment a variable inside a logging function call does not seem a good idea ...